core/.gitignore

```diff
@@ -41,6 +41,9 @@ next-env.d.ts
 # generated
 client/generated
 
+# next-intl
+messages/*.d.json.ts
```

Comment:
This file is generated by `next-intl` for strict typing of message arguments. Read more about it in the docs: https://next-intl.dev/docs/workflows/typescript#messages-arguments